oggm/cfg.py

```diff
@@ -182,6 +182,14 @@ def doc_str(self, key):
 _doc = ''
 BASENAMES['past_model'] = ('past_model.nc', _doc)
 
+_doc = 'Calving flux values'
+BASENAMES['calving_fluxes'] = ('calving_fluxes.pkl', _doc)
+
+_doc = 'Water depth'
+BASENAMES['water_depth'] = ('water_depth.pkl', _doc)
+
+_doc = 'Thickness at the calving front'
+BASENAMES['H_ice'] = ('H_ice.pkl', _doc)
 
```

Comment:
Maybe it would be easier to put all these data in one single dictionary, leaving only one entry for BASENAMES